src/sqlparser/src/parser.rs

```diff
@@ -561,7 +561,7 @@ impl Parser {
     pub fn parse_function(&mut self, name: ObjectName) -> Result<Expr, ParserError> {
         self.expect_token(&Token::LParen)?;
         let distinct = self.parse_all_or_distinct()?;
-        let args = self.parse_optional_args()?;
+        let (args, order_by) = self.parse_optional_args()?;
```

Comment:
Do we need to check the function name before deciding whether `ORDER BY` is acceptable?

Reply:
I think it is ok to accept `ORDER BY` without checking the function name since Postgresql seems to do the same.

Reply:
The check can be done in binder.